Assets/VRM/UniGLTF/Scripts/IO/UnityPath.cs

```diff
@@ -44,6 +44,24 @@ public bool IsUnderAssetsFolder
             }
         }
 
+        public bool IsStreamingAsset
+        {
+            get
+            {
+                if (IsNull)
+                {
+                    return false;
+                }
+
+                if (!FullPath.StartsWith(Application.streamingAssetsPath))
```

Comment:
```suggestion
                return FullPath.StartsWith(Application.streamingAssetsPath+"/");
```

Reply:
👍 
I avoid string `+`, but this one is safer